packages/core/src/shared/vscode/env.ts

```diff
@@ -104,9 +103,9 @@ export function isRemoteWorkspace(): boolean {
 
 /** Returns true if OS is Windows. */
 export function isWin(): boolean {
-    if (isWeb()) {
-        return false
-    }
+    // if (isWeb()) {
+    //     return false
+    // }
```

Comment:
why is this commented out?

Reply:
I added that recently to avoid using `process` on web, but it might be more trouble than benefit. Because `isWin` is called before globals are initialized (thus `isWeb()` fails).

If `process` is polyfilled for web-mode, then this check can be dropped.